internal/connection_repository/sql_connection_locator.go

```diff
@@ -187,25 +194,34 @@ func (scm *SqlConnectionLocator) GetAllConnections(ctx context.Context, offset i
 
 	for rows.Next() {
 		var account domain.AccountID
+		var orgIdString sql.NullString
 		var clientId domain.ClientID
 		var dispatchersString sql.NullString
 
-		if err := rows.Scan(&account, &clientId, &dispatchersString, &totalConnections); err != nil {
+		if err := rows.Scan(&account, &orgIdString, &clientId, &dispatchersString, &totalConnections); err != nil {
 			logger.LogError("SQL scan failed.  Skipping row.", err)
 			continue
 		}
 
+		var orgId domain.OrgID
+		if orgIdString.Valid {
+			err = json.Unmarshal([]byte(orgIdString.String), &orgId)
```

Comment:
The org_id will be stored as a string in the database.  So instead of doing a `json.Unmarshal()`, you'll just need to assign `orgIdString.String`  to  `orgId`.